cluster-autoscaler/utils/deletetaint/delete_test.go

```diff
@@ -45,7 +45,40 @@ func TestMarkNodes(t *testing.T) {
 	assert.True(t, HasToBeDeletedTaint(updatedNode))
 }
 
+func TestSoftMarkNodes(t *testing.T) {
+	node := BuildTestNode("node", 1000, 1000)
+	fakeClient := buildFakeClient(t, node)
+	err := MarkDeletionCandidate(node, fakeClient)
+	assert.NoError(t, err)
+
+	updatedNode, err := fakeClient.Core().Nodes().Get("node", metav1.GetOptions{})
+	assert.NoError(t, err)
+	assert.False(t, HasToBeDeletedTaint(updatedNode))
```

Comment:
nit: what is this scenario intended to test? Looks like it may make sense to assert soft taint is on the node here?

Reply:
Bug spotted, I have fixed it